### PR TITLE
modify resource links

### PR DESCRIPTION
**Pull request by helenlly**
## What changes were proposed in this pull request?

modify resource links

## How was this patch tested?

test passed in local server

### CODE REVIEW

File: docs/docs/other-resource.md [hunk 1]
```diff
@@ -1,20 +1,34 @@
 
 ---
 
-** Videos:** 
+** Public Talks:**
```
Comment (jason-dai) [inline docs/docs/other-resource.md]: Change to Talks

File: docs/docs/other-resource.md [hunk 2]
```diff
@@ -1,20 +1,34 @@
...
+
+* [Spark Summit 2017: BigDL: Bringing Ease of Use of Deep Learning for Apache Spark](https://spark-summit.org/2017/events/bigdl-bringing-ease-of-use-of-deep-learning-for-apache-spark/) 
+[[Slides Download]](https://www.slideshare.net/databricks/bigdl-bringing-ease-of-use-of-deep-learning-for-apache-spark-with-jason-dai-and-radhika-rangarajan)
 
-* [Introduction to BigDL on Apache Spark* Part3 ](https://software.intel.com/en-us/videos/introduction-to-bigdl-on-apache-spark-part3)
+* [Spark Summit 2017: Deep Learning to Big Data Analytics on Apache Spark Using BigDL](https://spark-summit.org/2017/events/deep-learning-to-big-data-analytics-on-apache-spark-using-bigdl/) [[Slides Download]](https://www.slideshare.net/databricks/deep-learning-to-big-data-analytics-on-apache-spark-using-bigdl-with-xianyan-jia-and-yuhao-yang)
 
-* [Spark Summit East 2017: Building Deep Learning Powered Big Data](https://spark-summit.org/east-2017/events/building-deep-learning-powered-big-data/)
+---
```
Comment (jason-dai) [inline docs/docs/other-resource.md]: What's the order for these?

File: docs/docs/other-resource.md [hunk 3]
```diff
@@ -1,20 +1,34 @@
...
+
+** Intel Resource:**
```
Comment (jason-dai) [inline docs/docs/other-resource.md]: Change to Others

File: docs/docs/other-resource.md [hunk 4]
```diff
@@ -1,20 +1,34 @@
...
 
-* [Spark Summit East 2017: Building Deep Learning Powered Big Data](https://spark-summit.org/east-2017/events/building-deep-learning-powered-big-data/)
+---
+
+** Intel Resource:** 
+
+* [BigDL Overview](https://software.intel.com/en-us/videos/bigdl-overview) 
 
-* [Spark Summit East 2017: BigDL: A Distributed Deep Learning Library on Spark](https://spark-summit.org/east-2017/events/bigdl-a-distributed-deep-learning-library-on-spark/)
+* [BigDL: Bring Deep Learning to the Fingertips of Big Data Users and Data Scientists](https://software.intel.com/en-us/articles/bigdl-bring-deep-learning-to-big-data-users-and-data-scientists)
```
Comment (jason-dai) [inline docs/docs/other-resource.md]: Remove this

File: docs/docs/other-resource.md [hunk 5]
```diff
@@ -1,20 +1,34 @@
...
 
-* [Spark Summit 2017: BigDL: Bringing Ease of Use of Deep Learning for Apache Spark](https://spark-summit.org/2017/events/bigdl-bringing-ease-of-use-of-deep-learning-for-apache-spark/)
+* [Introduction to BigDL on Apache Spark* Part1](https://software.intel.com/en-us/videos/introduction-to-bigdl-on-apache-spark-part1) 
+
+* [Introduction to BigDL on Apache Spark* Part2 ](https://software.intel.com/en-us/videos/introduction-to-bigdl-on-apache-spark-part2)
 
-* [Spark Summit 2017: Deep Learning to Big Data Analytics on Apache Spark Using BigDL](https://spark-summit.org/2017/events/deep-learning-to-big-data-analytics-on-apache-spark-using-bigdl/)
\ No newline at end of file
+* [Introduction to BigDL on Apache Spark* Part3 ](https://software.intel.com/en-us/videos/introduction-to-bigdl-on-apache-spark-part3)
```
Comment (jason-dai) [inline docs/docs/other-resource.md]: Combine these three to one bullet

File: docs/docs/other-resource.md (hunk 1 above)
Comment (helenlly) [reply docs/docs/other-resource.md]: done

File: docs/docs/other-resource.md (hunk 3 above)
Comment (helenlly) [reply docs/docs/other-resource.md]: done

File: docs/docs/other-resource.md (hunk 4 above)
Comment (helenlly) [reply docs/docs/other-resource.md]: done

File: docs/docs/other-resource.md (hunk 5 above)
Comment (helenlly) [reply docs/docs/other-resource.md]: the original video in intel.com are listed as 3 videos,   so I combine them into one summary bullet.   Is it ok? or Do you mean combining 3 videos into 1 video?

File: docs/docs/other-resource.md (hunk 2 above)
Comment (helenlly) [reply docs/docs/other-resource.md]: Strata first , then Spark Summit

**Pull request merged**